src/EditorFeatures/Core/Shared/Extensions/GlyphExtensions.cs

```diff
@@ -15,6 +15,9 @@ internal static class GlyphExtensions
         // https://github.com/dotnet/roslyn/issues/26642
         private static readonly Guid ImageCatalogGuid = Guid.Parse("ae27a6b0-e345-4288-96df-5eaf394ee369");
 
+        public static ImageId GetKnownImageId(int imageId)
```

Comment:
>GetKnownImageId [](start = 30, length = 15)

Why do we need this just for the feature? How did we do without it?

Reply:
This is just to avoid copying the guid for image catalog